app/controlplane/pkg/biz/workflowcontract.go

```diff
@@ -299,6 +300,47 @@ func (uc *WorkflowContractUseCase) Update(ctx context.Context, orgID, name strin
 	return c, nil
 }
 
+func (uc *WorkflowContractUseCase) ValidateContractPolicies(rawSchema []byte, token string) error {
```

Comment:
not required in this PR but it would be good to add some tests on this biz method

Reply:
I will do it.